s3.tf

```diff
@@ -1,7 +1,32 @@
 resource "aws_s3_bucket" "demo-example" {
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure versioning MFA is enabled for AWS S3 Buckets**
 
**Vulnerability**
Enabling MFA delete for versioning is a good way to add extra protection to sensitive files stored in AWS S3 Buckets.
 
**Remediation**
Using the AWS s3api CLI, enable MFA Delete for the S3 buckets that fail this rule, for example: aws s3api put-bucket-versioning --bucket bucketname --versioning-configuration Status=Enabled,MFADelete=Enabled --mfa your-mfa-serial-number mfa-code.

source: accurics-devenv